src/Altinn.Platform/Altinn.Platform.Storage/Storage/Authorization/StorageAccessHandler.cs

```diff
@@ -0,0 +1,190 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Altinn.Authorization.ABAC.Xacml;
+using Altinn.Authorization.ABAC.Xacml.JsonProfile;
+using Altinn.Common.PEP.Authorization;
+using Altinn.Common.PEP.Configuration;
+using Altinn.Common.PEP.Constants;
+using Altinn.Common.PEP.Helpers;
+using Altinn.Common.PEP.Interfaces;
+using Altinn.Platform.Storage.Helpers;
+using Altinn.Platform.Storage.Interface.Models;
+using Altinn.Platform.Storage.Repository;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace Altinn.Platform.Storage.Authorization
+{
+    /// <summary>
+    /// AuthorizationHandler that is created for handling access to app.
+    /// Authorizes based om AppAccessRequirement and app id from route
+    /// <see href="https://docs.asp.net/en/latest/security/authorization/policies.html"/> for details about authorization
+    /// in asp.net core
+    /// </summary>
+    public class StorageAccessHandler : AuthorizationHandler<AppAccessRequirement>
```

Comment:
Should this be renamed to StorageAccessRequierement to make it more understandable maybe?

Reply:
I dont think so since we reuse it in app.